src/Storages/MergeTree/MergeTreeIndexFullText.cpp

```diff
@@ -378,6 +410,15 @@ bool MergeTreeConditionFullText::atomFromAST(
         else if (!token_extractor->supportLike() && (func_name == "like" || func_name == "notLike"))
             return false;
 
+        if (func_name == "has")
+        {
+            out.key_column = key_column_num;
+            out.function = RPNElement::FUNCTION_HAS;
+            out.bloom_filter = std::make_unique<BloomFilter>(params);
+            stringToBloomFilter(const_value.get<String>(), token_extractor, *out.bloom_filter);
```

Comment:
What if first argument would be constant? Should we add `"has"` to `if` above where `key_arg_pos` is checked?

Reply:
It is checked above before we get into this function, this argument const_value is constant that we get from KeyCondition.